api/app/invitationletter/api.py

```diff
@@ -57,21 +57,27 @@ def post(self):
         if not offer:
             return errors.OFFER_NOT_FOUND
 
-        global invitation_template
+        invitation_template = None
 
-        if offer.accommodation_award and offer.travel_award:
+        if offer.accommodation_award and offer.accepted_accommodation_award \
+                and offer.travel_award and offer.accepted_travel_award:
             invitation_template = db.session.query(InvitationTemplate).filter(
                 InvitationTemplate.event_id == offer.event_id).filter(
                 InvitationTemplate.send_for_both_travel_accommodation).first()
-        elif offer.travel_award:
+
+        elif offer.travel_award and offer.accepted_travel_award:
             invitation_template = db.session.query(InvitationTemplate).filter(
                 InvitationTemplate.event_id == offer.event_id).filter(
                 InvitationTemplate.send_for_travel_award_only).first()
-        elif offer.accommodation_award:
+
+        elif offer.accommodation_award and offer.accepted_accommodation_award:
             invitation_template = db.session.query(InvitationTemplate).filter(
                 InvitationTemplate.event_id == offer.event_id).filter(
                 InvitationTemplate.send_for_accommodation_award_only).first()
 
+        elif (not offer.accommodation_award) and (not offer.travel_award):
+            return errors.NO_ACCOMMODATION_TRAVEL_AWARD
```

Comment:
It's not an error case. They also get a invitation letter even if they don't get an award. There is a default invitation letter

Reply:
so what's the condition to retrieve that template? 

> `....filter( not InvitationTemplate.send_for_both_travel_accommodation).filter( not InvitationTemplate.send_for_travel_award_only).filter( not InvitationTemplate.send_for_accommodation_award_only)` ?

Reply:
All things false :)